### ISSUE TITLE
RFE: use name from theming in authentication code message

### ISSUE THREAD

**Issue by putt1ck** [nextcloud/twofactor_gateway#274]

**Comment by ChristophWurst** [comment]
Yes! I've briefly mentioned this at https://github.com/nextcloud/twofactor_gateway/pull/253#discussion_r308589512 but we should indeed do this!

The only concern right now is that theming isn't properly exposed via a public API in the Nextcloud server. But I'll open a ticket for it if it doesn't already exist.

**Comment by ChristophWurst** [comment]
https://github.com/nextcloud/server/issues/15815 was the ticket
